data/noiseprofiles.json

```diff
@@ -3820,6 +3820,51 @@
       ]
     },
     {
+      "maker": "Pentax",
+      "models": [
+        {
```

Comment:
As CI tells you, that is not the right way to include noiseprofile.
You need to find already existing `"maker": "Pentax",` section,
and insert inner model (`{ "comment": "k-1 contributed b ... ]`) e.g. before `"model": "K-S2"` entry.